python/ray/data/tests/test_dataset_formats.py

```diff
@@ -2802,7 +2794,9 @@ def test_image_folder_datasource_e2e(ray_start_regular_shared):
     from torchvision.models import resnet18
 
     root = os.path.join(os.path.dirname(__file__), "image-folder")
```

Comment:
We should actually test whether the images have been resized right?

If the images in `image-folder` are all the same size to begin with, then this test will still pass even if resizing is not taking place.

Reply:
Good call. Fixed.